src/main/java/org/opensearch/ad/model/Entity.java

```diff
@@ -278,7 +278,7 @@ private static String normalizedAttributes(SortedMap<String, String> attributes)
     */
     public static Optional<String> getModelId(String detectorId, SortedMap<String, String> attributes) {
         if (attributes.isEmpty()) {
-            return Optional.empty();
+            return Optional.of(detectorId);
```

Comment:
prefer to use rcf partition 0 as we discussed.  Or you can leave it to me to change.

Reply:
it's up to you.